MaxText/sequence_packing.py

```diff
@@ -171,7 +171,7 @@ def true_fn():
       for k in keys:
         new_seq = one_example[k][: key2length[k]]
         new_seq_len = tf.size(new_seq)
-        new_partial[k] = tf.concat([partial[k], new_seq], pad_id)
+        new_partial[k] = tf.concat([partial[k], new_seq], 0)
```

Comment:
What was the error you were seeing because of this?

Reply:
I misunderstood this function in my previous PR. tf.concat is expecting an axis number here. So for deepseek tokenizer that defines pad_id as 100001, you got this error: "ValueError: Shape must be at least rank 100002 but is rank 1"